pyodata/v2/response.py

```diff
@@ -0,0 +1,32 @@
+"""
+Utility class to standardize response
+
+Author: Alberto Moio <email Alberto>, Nunzio Mauro <mnunzio90@gmail.com>
+Date:   2017-08-21
+"""
+import json
+
+
+class Response:
+    """Representation of http response in a standard form already used by handlers"""

```

Comment:
Odata v2 has both JSON and XML Atom as standard response...
https://www.odata.org/documentation/odata-version-2-0/atom-format/